dataset/dictionary/y-events.json5

```diff
@@ -1,12 +1,14 @@
 [
   {
-    en: "Theater Machanicus",
+    en: "Theater Mechanicus",
```

Comment:
Can you add the redirect setting to [redirect/words.json](https://github.com/xicri/genshin-langdata/blob/main/dataset/redirect/words.json) as well?

The current URL is https://genshin-dictionary.com/ja/theater-machanicus/, so you can configure the redirect by adding the following line:

```json
"theater-machanicus": "theater-mechanicus"
```